portal/src/main/webapp/js/src/DataProxyFactory.js

```diff
@@ -41,7 +41,7 @@ var DataProxyFactory = (function()
 	 *
 	 * @return singleton instance of MutationDataProxy class
 	 */
-	function getDefaultMutationDataProxy()
+	function getDefaultMutationDataProxy(_isVirtualStudy)
```

Comment:
Just a minor styling note: For function parameter names, it might be better not to start with underscore